playbooks/container/docker-rootless.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Since this role depends on docker being installed, suggest you add the dependency here, i.e.

```
---
- include: ../container/docker.yml
```

Reply:
No, there is not dependency on docker being installed. Rootless docker installs its own docker.
```
$ which docker
/sw/software/rootless-docker/bin/docker
```